packages/sdk-react/src/core/context/MoniteProvider.tsx

```diff
@@ -135,6 +135,10 @@ const createQueryClient = (i18n: I18n, sentryHub: Hub | undefined) =>
 
             return;
           }
+
+          toast.error(message, {
```

Comment:
Didn't we decide it was better not to handle all errors in this manner? Maybe I'm missing something in the implementation.

Reply:
It was missed part of the code. You can see that it is already implemented for **mutationCache**. I added it for **queryCache**.

We are handling onError event in one place instead of handling it in each query. **onError** callback was deprecated in v5 https://tanstack.com/query/latest/docs/framework/react/guides/migrating-to-v5#callbacks-on-usequery-and-queryobserver-have-been-removed

Reply:
As I remember, we decided to remove all onError and onSuccess callbacks from all queries and mutations.

Reply:
fixed

Removed global error handling.

Placed **useEffect** with error handling in places for **useQueries** that had it before migration to v5.